cmd/containerd-shim-runc-v2/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/containerd/containerd/v2/cmd/containerd-shim-runc-v2/manager"
 	_ "github.com/containerd/containerd/v2/cmd/containerd-shim-runc-v2/task/plugin"
 	"github.com/containerd/containerd/v2/pkg/shim"
+	_ "github.com/containerd/containerd/v2/pkg/tracing/plugin"
```

Comment:
I wonder how this affects the output binary size?

Context: https://github.com/containerd/containerd/pull/9233#discussion_r1358960606

Reply:
Looks like ~2MB larger with this import :(

Reply:
I'm fine leaving this out (or gated by a tag for debugging purposes?).
Mainly added it to see things working and found it didn't really seem to have an impact in memory usage so I kept it in.

Reply:
Seems like its easy to hide it with a build tag.

Reply:
Eventually we should find a way to drop grpc dependency from the runc shim.

Reply:
Gated this by the `shim_tracing` tag.